valid-anagram/oyeong011.cpp

```diff
@@ -0,0 +1,15 @@
+class Solution {
+public:
+    bool isAnagram(string s, string t) {
+        unordered_map<char, int> ss, tt;
+        if(s.length() != t.length())return false;
```

Comment:
```suggestion
        if(s.length() != t.length())return false;
        unordered_map<char, int> ss, tt;
```
s와 t의 길이가 다르면 unordered_map을 할당할 필요도 없으니까 5번 줄이 더 위에 있는게 나을 것 같아요